include/psen_scan_v2/scanner_controller.h

```diff
@@ -50,6 +51,44 @@ static constexpr uint32_t DEFAULT_SEQ_NUMBER{ 0 };
 
 using std::placeholders::_1;
 
+using namespace std::chrono_literals;
+class TimeoutTimer
+{
+public:
+  TimeoutTimer(const std::chrono::high_resolution_clock::duration& timeout,
+               const std::function<void()>& timeout_handler,
+               const std::chrono::high_resolution_clock::duration& start_timeout = 10ms)
+    : timer_thread_([this, timeout, timeout_handler]() {
+      thread_startetd_barrier_.release();
+      while (!terminated_ && !barrier_.waitTillRelease(timeout))
```

Comment:
I think the variable `terminate_` is superfluous.

Reply:
:smile: Yes, thx.

Reply:
Done.